map-prototype/index.html

```diff
@@ -0,0 +1,217 @@
+        });
+        map.mapTypes.set('map_style', styledMap);
+        map.setMapTypeId('map_style');
+
+        map.data.setStyle(function(feature) {
+          var status = feature.f.status;
+          var fillColor = (status == 'Installed') ? 'red' : 'gray';
+          var opacity = (status == 'Installed') ? .9 : .5;
+          var zIndex = (status == 'Installed') ? 100 : 1;
+          var scale = 8;
+          var strokeWeight = 1;
+          // supernode
+          if (feature.f.id == 227) {
+            scale = 10;
+            strokeWeight = 2;
+          }
+
+          return {
+            icon: {
+              path: google.maps.SymbolPath.CIRCLE,
+              fillColor: fillColor,
+              fillOpacity: opacity,
+              strokeOpacity: opacity,
+              scale: scale,
+              strokeColor: 'white',
+              strokeWeight: strokeWeight,
+              clickable: true,
+            },
+            //title: 'Node #'+feature.f.id,
+            title : feature.f.host,
+            zIndex: zIndex
+          }
+        });
+
+        map.data.loadGeoJson('./nodes.json');
+
+        var infowindow = new google.maps.InfoWindow();
+
+
+        map.data.addListener('click', function(event) {
+          var content = "<div class='pv3 tl'>"
+          content += "<p class='f4 fw7 di pr2 tl'>"+event.feature.f.host+"</p>"
+          content += "<br>"
+          if (event.feature.f.status == 'Installed') {
+            content += "<p class='di green f5 fw4'>Active</p>"
+          }else {
+            content += "<p class='di gray f5 fw4'>Potential</p>"
+          }
+          content += "<br>"
+          content += "<p class='di gray f5 fw4'>IPv6: "+event.feature.f.ipv6+"</p>"
+          content += "<br>"
+          content += "<p class='di blue f5 fw4'><a href='https://www.fc00.org/#"+event.feature.f.ipv6+"'>View on fc00.org</a></p>"
+          content += "</div>"
+
+          var panoramas = event.feature.f.panoramas
+          if (panoramas) {
+            content += "<p class='pv0 mv0 near-black fw4 tl'>View from this location:</p>"
+            for (var i = 0; i < panoramas.length; i++) {
+              var image = "<a href='../panorama/"+panoramas[i]+"'>"
+              image += "<div class='bg-near-white h5 w6 cover pv0 mt1 mb3' style='background-image: url(\"../panorama/"+panoramas[i]+"\")'></div>"
+              image += "</a>"
+
+              content += image;
+            }
+          }
+           infowindow.setContent(content);
+           infowindow.setPosition(event.feature.getGeometry().get());
+           infowindow.setOptions({pixelOffset: new google.maps.Size(-1,0)});
+           infowindow.open(map);
+        });
+
+        map.addListener("click", function(event) {
+          infowindow.close();
+        })
+
+
+      }
+    </script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBNClp7oJsw-eleEoR3-PQKV23tpeW-FpE&libraries=visualization&callback=initMap"
```

Comment:
I'd be up for working with a different library like leaflet (http://leafletjs.com/examples/geojson/) for some of this mapping work, we can work with a wider array of basemaps (not just google) if so.